File: app/src/main/java/com/frankegan/verdant/fullscreenimage/FullscreenImageContract.java
package com.frankegan.verdant.fullscreenimage;

/**
 * Created by frankegan on 5/18/16.
 */
public interface FullscreenImageContract {
    interface View {

        void setImage(String link);

        void setGif(String link);
    }

    interface UserActionsListener {

        void lightsOut();
    }
}

File: app/src/main/java/com/frankegan/verdant/fullscreenimage/FullscreenImageContract.kt
package com.frankegan.verdant.fullscreenimage

/**
 * Created by frankegan on 5/18/16.
 */
interface FullscreenImageContract {
    interface View {

        fun setImage(link: String)

        fun setGif(link: String)
    }

    interface UserActionsListener {

        fun lightsOut()
    }
}
